### PR TITLE
Add Shortcuts

### PR DESCRIPTION
**Pull request by naweidner**
- added overview page for shortcuts
- added shortcuts
- added translations for shortcuts

I'm not quite sure how intuitive the implemented shortcuts for Studio are. 
Please let me know if you know better/intuitive shortcuts or if you miss some.

### CODE REVIEW

File: src/ui/studio/keyboard-shortcuts/globalKeys.js
Comment (LukasKalbertodt) [inline src/ui/studio/keyboard-shortcuts/globalKeys.js:47]: Quite a bit of duplication in this array. Which is not a huge deal, but one could certainly make this smaller. An example:

```
const recordShortcuts = {
  RECORD_DISPLAY: {
    name: 'record-display',
    sequences: ['d'],
    description: ['d'],
  },
  // ...
};
const editShortcuts = /* ... */;

const mapValues = (obj, mapper) => Object.fromEntries(Object.entries(obj).map(mapper));

export const keyMap = {
  ...mapValues(recordShortcuts, ([key, value]) => 
    [key, { ...value, action: 'keydown', group: 'record-shortcuts' }]),
  ...mapValues(editShortcuts, ([key, value]) => 
    [key, { ...value, action: 'keydown', group: 'edit-shortcuts' }]),
  // ...
};
```

Whether thats really better ... :shrug: you decide

**Review by LukasKalbertodt — CHANGES_REQUESTED** [review]
Sorry for the long delay in reviewing this! I also only managed to do a quick review and test. But better than nothing!

I left a few inline comments, but I have additional comments from testing it. In random order:

- Shortcut overview page
    - The shortcut overview is quite "large" and somewhat breaks on screens widths < 480px. Maybe reduce the space between entries and also reduce the padding in the div that displays the key?
    - Not sure `<th>` is the right choice for these headings. 
    - Not sure `<table>` is the right choice there, given that you use `display: flex` anyway. I think screenreader and stuff doesn't treat it as a table anymore then, so you can also just use a `<div>`?
    - Some entries have three columns in total. Not sure I would do it like that. In particular, it would be nice if for example the "shift del" part of "Delete left crop marker" could could break into a new line on small screens, which is not currently possible.

- Choice of shortcuts
    - Alternative: use 1, 2, 3 for the choices (video and audio source). Then there are fewer different shortcuts.
    - Maybe show the shortcut on the buttons when pressing "alt"? Like Windows does in many programs.
    - ctrl+enter is already used by browser to open a selected link in a new tab, so it doesnt work in chrome. Neither does ctrl+backspace.
    - Cut: maybe pos1 and end?
    - Left/right arrow keys for next/back buttons


- After selecting camera, I'm not sure how to proceed with shortcuts alone (without tab).

- Menu at top is very wide now, already breaking for screen sizes <660px, which is problematic. I don't have a good fix for this except maybe putting the links to shortcuts (and info?) into settings? But I guess the proper solution is redesign of that section? Unfortunately, the design proposed by Lisa does not really address that. Except for removing the "recording" entry. Maybe one could remove the label and only show the icon for small screen sizes? But then we certainly want toolstips with the label instead.

File: src/i18n/locales/de.json
Comment (naweidner) [reply src/i18n/locales/de.json]: Changed it to "Shift" in german translation.

File: src/ui/studio/save-creation/index.js
Comment (naweidner) [reply src/ui/studio/save-creation/index.js]: Done. 
Removed `allowChanges={true}` from component and added `ignoreKeymapAndHandlerChangesByDefault: false` to keymap config. Since there are now multiple keyMaps with different names, I only pass the needed keyMap.

**Review by LukasKalbertodt — CHANGES_REQUESTED** [review]
Generally good changes, but I now reviewed in a bit more detail and found quite a few things that should still be fixed. For some style things (like missing semicolon or wrong spacing), I haven't marked every occurrence, so it would be great if you could go over the changes again and add missing semicolons for example. Same goes for the `getElementById` (see relevant comment below). It would also be great if commit message of new commits could follow the rules we recently talked about internal (i.e. [this blog post](https://chris.beams.io/posts/git-commit/)).

Sorry if many of these comments seem very nitpicky, but I do think we should stick to the code style already present. Though I agree Studio could use a better lint-config and/or formatter so that following the style is easier (and CI can flag violations automatically).

I have tested briefly and everything seems to work! I do need a bit more testing before merging this, but that can wait for the next round of reviews.

**Review by LukasKalbertodt — APPROVED** [review]
My comments were all addressed as far as I can tell. So I think it's finally time to merge this :D

**Pull request merged**
